spring-boot/src/main/appengine/app.yaml.example

```diff
@@ -1,13 +1,13 @@
 runtime: java
 env: flex
-service: application-monitoring-java
+service: <staging-springboot>
```

Comment:
this part is unclear -- does `<staging-springboot>` need to be replaced with something else in the actual app.yaml? could make it more explicit like, `<staging springboot service name goes here>`

Reply:
Hey so staging-applicaiton-monitoring-springboot was too many characters, it got rejected by Google when the gcloud command was run.

So I tried out staging-springboot because that reminded me of our discussion about maybe dropping the 'application-monitoring' part, as all of our App Engine services are for Application Monitoring (not sure it will always be that way).

Should I shorten them all? I could, in all the app.yaml.example files.

My concern with writing something like `<staging springboot service name goes here>` is that then they might follow no naming convention whatsoever. So I see `<staging-springboot>` and things like `<application-monitoring-flask>` as good placeholders.

I think the caret `<` also calls it to fail because it's not a valid character, which is good, because people should realize they might be overwriting the Master or Staging instances. If it's their first time applying, they should be updating the name here. 

I am not in support of the idea of , "I want to clone it....then run a deploy command a minute later...and not have to think about anything". I'm fine with the user going and researching what an app.yaml is about. We give enough instruction with, "Make sure your app.yaml has valid values", I don't want to write a bunch of instructions about every field.
@cstavitsky